config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -204,6 +208,8 @@ presubmits:
         - runner.sh
         - /workspace/scenarios/kubernetes_e2e.py
         args:
+        - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
```

Comment:
I don't think you need --repo flag, that's from a deprecated wrapper and is equivalent to extra_refs

Reply:
i see. is $(PULL_REFS) still needed, or the framework will automatically pull the PR since this is a presubmit? if needed, how to specify it?

Reply:
k/k is by default checked out at your $GOPATH at $PULL_REFS

k/release is checked out at head (master)

Reply:
ok. removed PULL_REFS, PTAL